<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSettings</class>
 <widget class="QDialog" name="WBSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="bottle_new">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_clone">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_delete">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40000</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_winecfg">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>WineCfg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_ninewinecfg">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>NineWineCfg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_d3dsettings">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>D3D settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_cp">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Control panel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_explorer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Explorer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_regedit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>RegEdit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_reboot">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Reboot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bottle_winetricks">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Winetricks</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="bottles_box">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Bottles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QListWidget" name="bottles">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
